resources/dicts/thoughts/alive/general.json

```diff
@@ -978,21 +1002,10 @@
         ],
         "main_trait_constraint": [
             "lonesome"
-        ],
-        "main_status_constraint": [
-            "apprentice",
-            "medicine cat apprentice",
-            "mediator apprentice",
-            "warrior",
-            "medicine cat",
-            "mediator",
-            "deputy",
-            "leader",
-            "elder"
-        ]   
+        ]  
```

Comment:
These aren't really appropriate for kits, which is what the status constraints prevented, you should add the status constraints back into this group.

Reply:
ditto to my reply on the daring thoughts thing; kits can't be lonesome, it was replaced with shy in the kit traits dict